package.json

```diff
@@ -16,22 +16,25 @@
   "license": "ISC",
   "dependencies": {
     "axios": "0.18.0",
+    "bcrypt-nodejs": "0.0.3",
     "body-parser": "1.18.3",
+    "classnames": "^2.2.6",
     "express": "4.16.4",
     "express-session": "1.15.6",
     "history": "4.7.2",
     "material-design-icons": "3.0.1",
     "mongodb": "3.1.10",
     "mongoose": "5.3.14",
     "prop-types": "15.6.2",
+    "ramda": "0.26.1",
```

Comment:
My bad. I added ramda because I wanted to use the clone function to create a deep clone but I decided to leave it on a future PR.  I will remove this. Nice catch!